test.js

```diff
@@ -13,20 +12,20 @@ const endpoints = [
   {url: '/api/repos/dwyl', status_code: 200}
 ];
 
-endpoints.forEach((endpoint) => {
-  test('GET :: ' + endpoint.url + ' :: returns ' + endpoint.status_code, (t) => {
-    t.plan(2);
+afterAll(() => {
```

Comment:
I don't think they've seen `afterAll` before this point, a comment would be helpful